test/CMakeLists.txt

```diff
@@ -15,8 +15,8 @@ endif()
 # Ensure all necessary libraries are included
 include_directories("${TENSORFLOW_SOURCE_DIR}")
 include_directories("${TENSORFLOW_SOURCE_DIR}/tensorflow/lite")
+include_directories("${TENSORFLOW_SOURCE_DIR}/../tflite_build/flatbuffers/include/")
```

Comment:
I think we removed this line, because the directory no longer gets created in the build script.

In the new CMake we now use:
https://github.com/ESE-Peasy/PosturePerfection/blob/dfd75d60c40495fd17162187c7dc8655da76bccb/src/CMakeLists.txt#L40

Reply:
Must have snuck in with the merge. I'll remove it